README.md

```diff
@@ -1,56 +1,52 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Survey form: Admission trail project
 
-> Description the project.
+<!-- please update the readme file -->
+<!-- please make sure there are no duplicates on your css -->
```

Comment:
As requested in the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#project-requirements) and by the previous reviewer, you're supposed to add these comments as inline comments, not to your `README.md` file. Kindly, follow [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) on how to add inline comments to your codebase and finally remove all comments from your `README.md` file to keep your file clean.